### PR TITLE
Fix/build

### PR DESCRIPTION
**Pull request by makosblade**
This PR:
- Lets builds run [✅ ]
- Fixes compile on rehearsal [✅ ]
- adds rehearsal to the monorepo, removing it as a submodule [✅ ]
- Runs rehearsal tests on each run [❌ ]

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -6,8 +6,11 @@ logging/
...
 build/
+out/
 artifacts/
-gradle.properties
 .DS_Store
 *.iml
 *.swp
+conductor/src/main/resources/auth0.yaml
+datastore/src/main/resources/auth0.yaml
+indexer/src/main/resources/auth0.yaml
```
Comment (katherinebernstein) [inline .gitignore:16]: probably wouldn't hurt to add the linker one here too

**Review by katherinebernstein — APPROVED** [review]

File: conductor-client/build.gradle [hunk 2]
```diff
@@ -43,7 +43,7 @@ ext.scmUrl = 'scm:git:https://github.com/openlattice/conductor-client.git'
 ext.connectionUrl = 'scm:git:https://github.com/openlattice/conductor-client.git'
 ext.developerConnectionUrl = 'scm:git:https://github.com/openlattice/conductor-client.git'
 
-apply from: "../gradles/openlattice.gradle"
+apply from: "https://raw.githubusercontent.com/openlattice/openlattice/fix/build/gradles/openlattice.gradle"
```
Comment (UnsungHero97) [inline conductor-client/build.gradle:46]: Same comment about `fix/build` vs `develop`.

File: conductor/build.gradle [hunk 3]
```diff
@@ -45,7 +46,7 @@ ext.scmUrl = 'scm:git:https://github.com/openlattice/conductor.git'
 ext.connectionUrl = 'scm:git:https://github.com/openlattice/conductor.git'
 ext.developerConnectionUrl = 'scm:git:https://github.com/openlattice/conductor.git'
 
-apply from: "../gradles/openlattice.gradle"
+apply from: "https://raw.githubusercontent.com/openlattice/openlattice/fix/build/gradles/openlattice.gradle"
```
Comment (UnsungHero97) [inline conductor/build.gradle:49]: Same comment about `fix/build` vs `develop`.

File: courier/build.gradle [hunk 4]
```diff
@@ -38,28 +39,43 @@ ext.scmUrl = 'scm:git:https://github.com/openlattice/courier.git'
 ext.connectionUrl = 'scm:git:https://github.com/openlattice/courier.git'
 ext.developerConnectionUrl = 'scm:git:https://github.com/openlattice/courier.git'
 
-apply from: "../gradles/openlattice.gradle"
+apply from: "https://raw.githubusercontent.com/openlattice/openlattice/fix/build/gradles/openlattice.gradle"
```
Comment (UnsungHero97) [inline courier/build.gradle:42]: Same comment about `fix/build` vs `develop`.

File: datastore/build.gradle [hunk 5]
```diff
@@ -43,7 +43,7 @@ distTar {
     archiveName "datastore.tgz"
 }
 
-apply from: "../gradles/openlattice.gradle"
+apply from: "https://raw.githubusercontent.com/openlattice/openlattice/fix/build/gradles/openlattice.gradle"
```
Comment (UnsungHero97) [inline datastore/build.gradle:46]: Same comment about `fix/build` vs `develop`.

File: gradles/openlattice.gradle [hunk 6]
```diff
@@ -64,8 +60,8 @@ tasks.withType(Tar){
   compression = Compression.GZIP
 }
 
-apply from: "https://raw.githubusercontent.com/openlattice/gradles/master/repos.gradle"
-apply from: "https://raw.githubusercontent.com/openlattice/gradles/master/publish.gradle"
+apply from: "https://raw.githubusercontent.com/openlattice/openlattice/fix/build/gradles/repos.gradle"
+apply from: "https://raw.githubusercontent.com/openlattice/openlattice/fix/build/gradles/publish.gradle"
```
Comment (UnsungHero97) [inline gradles/openlattice.gradle:64]: Same comment about `fix/build` vs `develop`.

File: indexer/build.gradle [hunk 7]
```diff
@@ -55,7 +54,7 @@ ext.scmUrl = 'scm:git:https://github.com/openlattice/indexer.git'
 ext.connectionUrl = 'scm:git:https://github.com/openlattice/indexer.git'
 ext.developerConnectionUrl = 'scm:git:https://github.com/openlattice/indexer.git'
 
-apply from: "../gradles/openlattice.gradle"
+apply from: "https://raw.githubusercontent.com/openlattice/openlattice/fix/build/gradles/openlattice.gradle"
```
Comment (UnsungHero97) [inline indexer/build.gradle:57]: Same comment about `fix/build` vs `develop`.

File: linker/build.gradle [hunk 8]
```diff
@@ -55,7 +55,7 @@ ext.scmUrl = 'scm:git:https://github.com/openlattice/linker.git'
 ext.connectionUrl = 'scm:git:https://github.com/openlattice/linker.git'
 ext.developerConnectionUrl = 'scm:git:https://github.com/openlattice/linker.git'
 
-apply from: "../gradles/openlattice.gradle"
+apply from: "https://raw.githubusercontent.com/openlattice/openlattice/fix/build/gradles/openlattice.gradle"
```
Comment (UnsungHero97) [inline linker/build.gradle:58]: Same comment about `fix/build` vs `develop`.

File: rehearsal/build.gradle [hunk 9]
```diff
@@ -0,0 +1,114 @@
...
+    id 'signing'
+    id 'com.github.spotbugs' version '4.6.0'
+    id 'org.jetbrains.kotlin.jvm' version '1.3.72'
+    id 'org.jetbrains.kotlin.plugin.spring' version '1.3.72'
+}
+
+description = "Integration Tests"
+group = "com.openlattice"
+
+apply from: "https://raw.githubusercontent.com/openlattice/openlattice/fix/build/gradles/openlattice.gradle"
```
Comment (UnsungHero97) [inline rehearsal/build.gradle:35]: Same comment about `fix/build` vs `develop`.

File: rehearsal/gradle/wrapper/gradle-wrapper.properties [hunk 10]
```diff
@@ -0,0 +1,5 @@
+distributionBase=GRADLE_USER_HOME
+distributionPath=wrapper/dists
+distributionUrl=https\://services.gradle.org/distributions/gradle-5.6.4-bin.zip
```
Comment (UnsungHero97) [inline rehearsal/gradle/wrapper/gradle-wrapper.properties:3]: Any reason this is on v5 instead of v6?

File: scribe/build.gradle [hunk 11]
```diff
@@ -38,12 +39,23 @@ ext.scmUrl = 'scm:git:https://github.com/openlattice/scribe.git'
 ext.connectionUrl = 'scm:git:https://github.com/openlattice/scribe.git'
 ext.developerConnectionUrl = 'scm:git:https://github.com/openlattice/scribe.git'
 
-apply from: "../gradles/openlattice.gradle"
+apply from: "https://raw.githubusercontent.com/openlattice/openlattice/fix/build/gradles/openlattice.gradle"
```
Comment (UnsungHero97) [inline scribe/build.gradle:42]: Same comment about `fix/build` vs `develop`.

File: scrunchie/build.gradle [hunk 12]
```diff
@@ -20,12 +21,30 @@ ext.connectionUrl = 'scm:git:https://github.com/openlattice/scrunchie.git'
 ext.developerConnectionUrl = 'scm:git:https://github.com/openlattice/scrunchie.git'
 
 apply plugin: 'kotlin'
-apply from: 'https://raw.githubusercontent.com/openlattice/gradles/master/openlattice.gradle'
-
+apply from: "https://raw.githubusercontent.com/openlattice/openlattice/fix/build/gradles/openlattice.gradle"
```
Comment (UnsungHero97) [inline scrunchie/build.gradle:24]: Same comment about `fix/build` vs `develop`.

**Review by UnsungHero97 — APPROVED** [review]

File: settings.gradle.kts [hunk 13]
```diff
@@ -0,0 +1,39 @@
+pluginManagement {
+    plugins {
+        kotlin("jvm")                               version "1.3.72" apply false
+
```
[the remark this answers is not in the document]
Comment (makosblade) [reply settings.gradle.kts:4]: They're aligned in columns

**Review by katherinebernstein — APPROVED** [review]

**Pull request merged**